java/mykola-melnyk/src/com/zoolatech/lecture3/tasks/_1/musician/Musician.java

```diff
@@ -0,0 +1,50 @@
+package com.zoolatech.lecture3.tasks._1.musician;
+
+import com.zoolatech.lecture3.tasks._1.musical_instrument.*;
+
+import java.util.Objects;
+
+public class Musician {
+    private String name;
+    private MusicalInstrument instrument;
+    public Musician (String name, MusicalInstrument instrument) {
+        this.name = name;
+        this.instrument = instrument;
+        System.out.printf("Meet %s %s playing %s, which sounds like this: %s. \n",
+                getClass().getSimpleName(), name, instrument.getName(),
+                instrument.getSound());
+    }
+
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
1. if you override the method equals, don't forget to override hashCode as well
2. method equals is usually defined at the end of a class

Reply:
Added hashCode, moved equals and hashCode down.